### DISCUSSION TITLE
Does `pyright` have something against `find` literal? =)

### DISCUSSION THREAD

**Discussion by belegnar** [microsoft/pyright#8743] [Q&A] [answered]
Code sample in [pyright playground](https://pyright-play.net/?strict=true&code=JYWwDg9gTgLgBAUwHYFcQCh0GMA2BDAZwLgFFUQAKZNAOgGUYoy0BKALnTi7gIhARgALYEgDmcALxwARL35CRo6Z24AzEQBNJM9Ug3KgA)

```python
import enum

class Enum(enum.StrEnum):
    something = "something"
    find = "find"

```

**Comment by erictraut** [comment, marked as answer]
The type `StrEnum` derives from both `str` and `ReprEnum`. The `str` class declares a method called `find`, and you are attempting to overwrite this with a value whose type is incompatible with the existing declaration.
